.github/scripts/Update_Database.py

```diff
@@ -0,0 +1,117 @@
+from github import Github
+import json
+import sys
+import re
+
+# Regex Patterns
+category = r"- \[x\] (.+)"
+name = r"Title: (.+)"
+path = r"Folder: (.+)"
+requirments_path = r"Requirments: (.+)"
+entry = r"Script: (.+)"
+arguments = r"Arguments: (.+)"
+contributor = r"Contributor: (.+)"
+description = r"Description: (.+)"
+
+
+def add_script(category, name, path, entry, arguments, requirments_path, contributor, description, pa_token):
+    """ Add a Contributor script to database """
+    new_data = {category: {name: [path, entry, arguments, requirments_path, contributor, description]}}
+    data_store = read_data()
+    
+    try:
+        # If category doesn't exist try will fail and except will ask to add a new category with the project
+        if data_store[category]:                                          # Check for existing category or a new one
+                data_store[category].update(new_data[category])           # Add script
+    except:
+        data_store.update(new_data)                                       # Add new category
+
+    # <----- This part is to avoid a single/double quotes error when trying to update the database with PyGithub ----->
+    with open("./Master Script/datastore.json", "w") as file:
+        json.dump(data_store, file)
```

Comment:
add a indent parameter so that the JSON is more readable for users
:
![image](https://user-images.githubusercontent.com/43691873/117608846-7f9d2c00-b17c-11eb-8cd1-ec1810d5446a.png)

It will become like this:
![image](https://user-images.githubusercontent.com/43691873/117608902-98a5dd00-b17c-11eb-9589-a970b1ed906f.png)